File: src/2-Problem08.pas
program problem8;

var 
    A,B : integer;
    i : integer;
    j : integer;
    isPrima : boolean;
begin
    write('Masukan nilai A : '); readln(A);
    write('Masukan nilai B : '); readln(B);
    writeln('Bilangin prima diantera ',A,' sampai ',B,' adalah');
    
    for i:= A to B do
      begin
        
        isPrima := true;
        j := 2;
        
        while ( ( j <= Sqrt(i) ) and ( isPrima ) ) do
          begin 
            if (i mod j = 0 ) then
              begin
                isPrima := false;
              end;
            j := j + 1;
          end;

        if ( ( isPrima ) and ( i <> 1) ) then
            writeln(i);
      end;
end.

File: src/2-Problem08.py
#Nama	: Dandy Arif Rahman
#NIM	: 13516086
#Soal	: BAB 2 - Problem 8
from math import sqrt, floor

ListPrime = []
A = int(input("Masukkan nilai A :"))
B = int(input("Masukkan nilai B :"))
print("Bilangan prima antara %d sampai %d adalah" %  (A, B))

for num in range(2, B + 1):
	temp = floor(sqrt(num))
	prime = True
	for i in range(2, temp+1):	
		if num % i == 0:
			prime = False
			break
	if prime : ListPrime.append(num)
for prime in ListPrime:
	print(prime)